src/content/docs/apm/agents/nodejs-agent/installation-configuration/nodejs-agent-configuration.mdx

```diff
@@ -4108,6 +4108,47 @@ For tips on configuring logs for the Node.js agent, see [Configure Node.js logs
   </Collapser>
 </CollapserGroup>
 
+## Version tracking [#version-tracking]
+
+
+Setting `NEW_RELIC_METADATA_SERVICE_VERSION` creates a tag, `tag.service.version` on event data. This tag is the version of your deployed code, in many cases a semantic version such as 1.2.3, but not always. Sending this information allows you to facet your telemetry by the version of the software deployed, so you can quickly identify which versions of your software are producing the errors. 
+
+An upcoming release of errors inbox will automatically track which versions of your software are producing errors. 
+
+
+<CollapserGroup>
+  <Collapser
+    id="version-tracking-collapser"
+    title="version-tracking"
+  >
+    <table>
+      <tbody>
+        <tr>
+          <th>
+            Type
+          </th>
+
+          <td>
+            String
+          </td>
+        </tr>
+
+        <tr>
+          <th>
```

Comment:
Should we also include here that the default is nothing, like the other environment variables specify their defaults?

Reply:
I think we should if it's more clear. I'm going to add it.